spec/support/shared_examples/participant_action_support.rb

```diff
@@ -33,6 +33,16 @@
       it { is_expected.to have_error(:participant_id, :invalid_participant, "Your update cannot be made as the '#/participant_id' is not recognised. Check participant details and try again.") }
     end
   end
+
+  describe ".new_filtering_attributes" do
+    context "when extra params are sent to the action service" do
+      subject { described_class.new_filtering_attributes(lead_provider:, participant:, course_identifier:, reason: "test", schedule_identifier: "any") }
```

Comment:
I think this changed from `participant` to `participant_id` recently

```suggestion
      subject { described_class.new_filtering_attributes(lead_provider:, participant_id:, course_identifier:, reason: "test", schedule_identifier: "any") }
```